### ISSUE TITLE
spices-notifier@germanfr - Lists deprecated spices.

### ISSUE THREAD

**Issue by smurphos** [linuxmint/cinnamon-spices-applets#1963]
```
 * Applet version/Build date : 3.8
 * Cinnamon version : 3.8.9
 * Distribution - Mint 19
 * Graphics hardware *and* driver used
           Card: Intel 2nd Generation Core Integrated Graphics Controller
           Display Server: x11 (X.Org 1.19.6 )
           drivers: modesetting (unloaded: fbdev,vesa)
           Resolution: 1366x768@60.03hz
           OpenGL: renderer: Mesa DRI Intel Sandybridge Mobile
           version: 3.3 Mesa 18.0.5

 * Snippet or attachment of ~/.xsession-errors
`(cinnamon:1639): Cjs-WARNING **: 05:25:14.273: JS ERROR: TypeError: result is null
anonymous/get_comment_count/<@/home/steve/.local/share/cinnamon/applets/spices-notifier@germanfr/applet.js:195:9`

 ```

**Notify author of applet**
@germanfr 

**Issue**
Hi German,

The applet is picking up spices removed from the master repo - in my case the theme Cinnamox-Zanah. This occurs if if I remove the applet completely, delete `~/.cinnamon/configs/spices-notifier@germanfr` and then set up from fresh.

**Steps to reproduce**
Load the applet using my user name - 'smurphos'

**Expected behaviour**
Spices no longer in the Master branch should not be added to the list and ideally should be removed on the next refresh after removal from the Master branch.

**Other information**
I'm presuming the exception in .xsession-errors is due to the Cinnamon Spices page from Cinnamox-Zanah no longer existing.

**Comment by germanfr** [comment]
@smurphos the error is already fixed, but the issue behind the scenes, which is the request being cached forever, still persists; which also makes the star count to be outdated. If you know anything about fixing that please let me know. Besides that, this can be closed.

**Comment by smurphos** [comment]
Thanks - yep the theme.json file obtained via wget includes a reference to Zanah and the modification date is listed as 8th August 2018. 

If I open https://cinnamon-spices.linuxmint.com/json/themes.json directly in my browser it's absent.

[themes.json.zip](https://github.com/linuxmint/cinnamon-spices-applets/files/2393959/themes.json.zip)

**Comment by germanfr** [comment]
That happens to me too. I thought it was my computer caching the request, but it seems it's the server who caches depending on the user agent. So @smurphos, the JSON predates your computer installation?

**Comment by smurphos** [comment]
Yes it does. I can force an up to date copy this way though...

wget https://cinnamon-spices.linuxmint.com/json/themes.json?version=1

**Comment by germanfr** [comment]
Oh, thank you for the tip. I'll see if that works in the applet itself :wink:

**Comment by smurphos** [comment]
Fixed by #1967
